packages/palette/src/uiColors.ts

```diff
@@ -1,17 +1,7 @@
 export const white = '#FFFFFF';
 export const black = '#061621';
 
-interface HueRange {
```

Comment:
Why are we removing this?

Reply:
Having a declared type on the variables takes precedence over `as const`. This way, in the storybook (for instance) we don't need to include a fallback to `transparent` because it knows what keys are defined. I added the `palette.spec.ts` file with the thinking that if we go outside of the defined `HueRange` type it would throw a type error there, so we still get that consistency.

Reply:
That test doesn't protect us from adding a key outside of `HueRange`, btw, but it does at least ensure that we didn't remove any of the required hues, or set it to a number or something.

Reply:
Actually... maybe it'd be better to just do a regular test instead of trying to get TS to do it for us

Reply:
Alright I did that